.github/workflows/docs.yml

```diff
@@ -2,6 +2,9 @@ on:
   pull_request:
     branches:
       - main
+  push:
```

Comment:
This will make it easier to look through https://github.com/googleapis/python-bigquery-dataframes/actions?query=branch%3Amain+event%3Apush to see when failures started.